.github/workflows/ci.yml

```diff
@@ -0,0 +1,153 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - "release/**"
+env:
+  APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+  APPLITOOLS_BATCH_ID: ${{ github.sha }}
+  APPLITOOLS_BATCH_NAME: ${{ github.head_ref }}
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: actions/setup-node@v2
```

Comment:
this actions plugin uses actions/cache under the hood. It would be good to experiment here to see how our two levels of caching will or will not speed up CI builds. See: https://github.com/actions/setup-node#caching-packages-dependencies

Reply:
I gave it a try, seemed to make no noticeable difference from what I could tell.